### PR TITLE
Improve product detail - made description listed & made rating stars

### PR DESCRIPTION
**Pull request by robertpengcode**
_Your PR Notes Here_
changed product description in product detail page from a long paragraph to listed items.
made rating stars using CSS & JS. a reviewStars component was created.
made the box that contains reviews be able to scroll down (CSS).
added two more reviews in seed to show the review box can be scroll down.
removed several unused dependencies that installed while developing...

### CODE REVIEW

File: client/components/reviewStars.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react'
+
+const ReviewStars = ({rating}) => {
+  const convertToWidth = rating => {
+    return `${Math.round(rating / 5 * 100 / 10) * 10}%`
```
Comment (avc278) [inline client/components/reviewStars.js:5]: Could you add some parentheses to show what operations happen in what order for this calculation?

File: public/index.html [hunk 2]
```diff
@@ -16,6 +17,7 @@
   </head>
   <body>
     <div id="app"></div>
+
```
Comment (avc278) [inline public/index.html:20]: Let's undo this whitespace

Comment (robertpengcode) [reply public/index.html:20]: yes, removed.

File: client/components/reviewStars.js (hunk 1 above)
Comment (robertpengcode) [reply client/components/reviewStars.js:5]: ok, put parentheses there. that math thing gives us a string like '90%' '80%' '70%'...  but not '75%'

**Pull request merged**